datadog_checks_dev/datadog_checks/dev/tooling/commands/create.py

```diff
@@ -152,7 +152,26 @@ def create(ctx, name, integration_type, location, manifest_v2, non_interactive,
         else:
             # Fill in all common non Marketplace fields
             template_fields['pricing_plan'] = ''
-            template_fields['author_info'] = ''
+            if repo_choice == 'core':
+                template_fields[
+                    'author_info'
+                ] = """
+  "author": {
+    "support_email": "help@datadoghq.com",
+    "name": "Datadog",
+    "homepage": "https://www.datadoghq.com",
+    "sales_email": "info@datadoghq.com"
+  },"""
+            else:
+                template_fields[
+                    'author_info'
+                ] = """
+  "author": {
+    "support_email": "",
+    "name": "",
+    "homepage": "",
+    "sales_email": ""
```

Comment:
We ask for some of this in the workflow already. When I tested creating an integration through extras:

```
Email used for support requests: <input>
Your name: <input>
```

and similarly for marketplace:

```
Email used for support requests: <input>
Your Company Name: <input>
The product or company homepage: <input>
The Legal email used to receive subscription notifications: <input>
```

I think we should use the output of these to populate as much of the `author` fields as we can (or stop asking for those as input, but I have a preference for the former). With the way it is now, it creates a manifest where all author fields are just empty strings.